modules/core/04-channel/types/msgs_test.go

```diff
@@ -55,6 +55,8 @@ var (
 	packet        = types.NewPacket(validPacketData, 1, portid, chanid, cpportid, cpchanid, timeoutHeight, timeoutTimestamp)
 	invalidPacket = types.NewPacket(unknownPacketData, 0, portid, chanid, cpportid, cpchanid, timeoutHeight, timeoutTimestamp)
 
+	emptyProof = []byte{}
```

Comment:
Would it make sense to also have tests for a nil proof?

Reply:
I think its probably overkill. We check `len(proof) != 0` and the built-in `len` function treats `[]byte{}` and `nil` the same.